### DISCUSSION TITLE
安装MMpretrain出现这个错误怎么解决啊？

### DISCUSSION THREAD

**Discussion by Moon-creat** [open-mmlab/mmclassification#1703] [Q&A]
![image](https://github.com/open-mmlab/mmpretrain/assets/129244270/4fd9d04d-59c7-463d-b80b-f543a8268e30)

**Comment by Ezra-Yu** [comment]
run 
```
pip install -U openmim
```

![image](https://github.com/open-mmlab/mmpretrain/assets/18586273/d57f5a70-a894-44bb-8ead-35ef9bdabda3)
